src/main/java/com/mattmerr/beets/commands/PlayCommand.java

```diff
@@ -56,10 +62,17 @@ public Mono<Void> execute(SlashCommandEvent event) {
     logCall(event);
     String beet = asRequiredString(event.getOption("beet"));
     Snowflake guildId = requireGuildId(event.getInteraction());
+
+    var multiqueue = asBoolean(event.getOption("multiqueue")).orElse(false);
     var delayMinutes = asLong(event.getOption("delay"));
-    if (delayMinutes.isPresent() && (delayMinutes.get() < 1 || delayMinutes.get() > 120)) {
-      return event.reply("Delay must be in range [1, 120] minutes");
+    if (delayMinutes.isPresent()) {
```

Comment:
I sort of prefer just adding the check and repeating .isPresent() call to reduce nesting
```
if (delayMinutes.isPresent() && multiqueue) {
  return event.reply("You cannot enable multiqueue and delay at the same time");
}
if (delayMinutes.isPresent() && .....
```

Reply:
Done.